src/Libraries/Nop.Services/Installation/InstallationService.cs

```diff
@@ -2834,9 +2834,9 @@ protected virtual async Task InstallSettingsAsync(RegionInfo regionInfo)
 
             await settingService.SaveSettingAsync(new AdminAreaSettings
             {
-                DefaultGridPageSize = 15,
-                PopupGridPageSize = 7,
-                GridPageSizes = "7, 15, 20, 50, 100",
+                DefaultGridPageSize = 20,
+                PopupGridPageSize = 10,
+                GridPageSizes = "10, 20, 50, 100",
```

Comment:
ok, now lets do more interesting, can you try to make sure GridPageSzies was changed just for Product Listing and not for the whole Admin?